source/uwp/Renderer/lib/AdaptiveTextBlockRenderer.cpp

```diff
@@ -89,4 +115,52 @@ namespace AdaptiveNamespace
             jsonObject, elementParserRegistration, actionParserRegistration, adaptiveWarnings, element);
     }
     CATCH_RETURN;
+
+    HRESULT AdaptiveTextBlockRenderer::GetHeadingLevelFromContext(ABI::AdaptiveNamespace::IAdaptiveRenderContext* renderContext,
+                                                                  ABI::Windows::UI::Xaml::Automation::Peers::AutomationHeadingLevel* headingLevel)
+    {
+        ComPtr<IAdaptiveHostConfig> hostConfig;
+        RETURN_IF_FAILED(renderContext->get_HostConfig(&hostConfig));
```

Comment:
nit: have you considered moving this method to RenderContext since it owns the host config, and we don't have to pass in the render context?

Reply:
That's not a bad idea, except the IAdaptiveRenderContext interface is public. I don't really want to put this there i don't think since it feels like an internal implementation detail rather than something we want to commit to the public API. I could put it on the implementation class only and use "Peek" to get at it, but at that point i think it's probably more trouble than it's worth.

---
In reply to: [606039737](https://github.com/microsoft/AdaptiveCards/pull/5612#discussion_r606039737) [](ancestors = 606039737)